### ISSUE TITLE
Ability to power up weapons in WPN pages

### ISSUE THREAD

**Issue by NikolaiVChr** [NikolaiVChr/f16#558]
This requires rewrite of station-management.nas

**Comment by NikolaiVChr** [comment]
For Maverick, real startup time is 180 seconds, with battery time ranging from 30 to 60 minutes depending on video use. Takes an hour to recharge.
Setting battery time to 45 minutes.
For AGM-88, 1200 seconds of battery time is estimated.

**Comment by NikolaiVChr** [comment]
61df2df30321398646f16d2c0afad9e916122a0c enables battery consumption

**Comment by JMaverick16** [comment]
> For Maverick, real startup time is 180 seconds, with battery time ranging from 30 to 60 minutes depending on video use. Would it make sense to have 65B variant with 30min battery while 65D with 60min?

**Comment by NikolaiVChr** [comment]
Not really, unless you forbid people to use TGP with the B. (as we simulate both weapon video and tgp with tgp)
Its now an average of 45 mins for both.
